tests/overlays/mod.rs

```diff
@@ -1408,3 +1408,50 @@ fn overlay_help_no_error() {
     let actual = nu!("overlay use -h");
     assert!(actual.err.is_empty());
 }
+
+#[test]
+fn test_overlay_use_with_printing_file_pwd() {
+    Playground::setup("use_with_printing_file_pwd", |dirs, nu| {
+        let file = dirs.test().join("foo").join("mod.nu");
+        nu.mkdir("foo").with_files(&[FileWithContent(
+            file.as_os_str().to_str().unwrap(),
+            r#"
+                export-env {
+                    print $env.FILE_PWD
+                }
```

Comment:
I ain't no overlay god like @kubouch , but why is this wrapped in a `export-env` block when it just reads the variable and doesn't export it.

Reply:
The block is run with `overlay use`, so it's a way how to print the env vars from within the module.

Reply:
Ah right this forces evaluation, thanks for the explanation!